tests/unittest/_torch/multi_gpu/test_moe_a2a_rank_mask.py

```
@@ -0,0 +1,407 @@
+            f"Need at least {ep_size} GPUs but only {torch.cuda.device_count()} are available"
+        )
+
+    hidden_size = 1024
+    num_experts = 32
+    workspace_size_per_rank = 256 * 1024 * 1024
+
+    args = (ep_size, local_num_tokens, top_k, workspace_size_per_rank, num_experts, hidden_size)
+    results = list(
+        mpi_pool_executor.map(
+            _worker_all_active_matches_no_mask,
+            *zip(*[args] * ep_size),
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)